Doc/library/datetime.rst

```diff
@@ -1347,22 +1347,22 @@ Instance methods:
 
    where ``yday = d.toordinal() - date(d.year, 1, 1).toordinal() + 1``
    is the day number within the current year starting with ``1`` for January
-   1st. The :attr:`tm_isdst<time.struct_time.tm_isdst>` flag of the result is set according to the
+   1st. The :attr:`~.time.struct_time.tm_isdst` flag of the result is set according to the
```

Comment:
I think that a dot is not needed here.
```suggestion
   1st. The :attr:`~time.struct_time.tm_isdst` flag of the result is set according to the
```

Reply:
I thought the dot was needed (I didn't think of that on my own) to keep Sphinx from confusing `datetime.time` objects with the `time` module. I think Hugo pointed out that construct. I'm going to leave this as-is.

Reply:
https://www.sphinx-doc.org/en/master/usage/domains/python.html
> Normally, names in these roles are searched first without any further qualification, then with the current module name prepended, then with the current module and class name (if any) prepended. If you prefix the name with a dot, this order is reversed.

If I understood it correctly, for `.time.struct_time.tm_isdst` the order of searches:
* `datetime.time.struct_time.tm_isdst` -- fails
* `time.struct_time.tm_isdst` -- found

For `time.struct_time.tm_isdst` the order of searches:
* `time.struct_time.tm_isdst` -- found
* `datetime.time.struct_time.tm_isdst` -- no need to search

So it found the same result in both cases, but without a dot if finds this reference faster.